site/src/main/java/uk/nhs/digital/nil/beans/Indicator.java

```diff
@@ -85,7 +86,7 @@ public boolean hasAttachments() {
         String ASSUREDSTATUS = "nationalindicatorlibrary:assuredStatus";
         String PUBLISHEDBY = "nationalindicatorlibrary:publishedBy";
         String REPORTINGLEVEL = "nationalindicatorlibrary:reportingLevel";
-        String GEOGRAPHIC_COVERAGE = "publicationsystem:GeographicCoverage";
```

Comment:
Will you need to update all the existing indicators if you change this?

Reply:
as discussed, left as publicationsystem:GeographicCoverage